### ISSUE TITLE
Add OPML Support

### ISSUE THREAD

**Issue by Cralex** [Wowfunhappy/Pebble-RSS-Reader#12]
This is probably more helpful when testing updates, but I’d like the ability to save my feeds to a file that I can download to my phone, and then upload later if needed. (Can pebble app settings do that? Maybe a window showing a block of text that could be copied from, hidden behind a debug or developer option would suffice.)

**Comment by Wowfunhappy** [comment]
There's a standard format for importing/exporting lists of RSS feeds—[OPML](http://dev.opml.org). This would also give us interoperability with other clients. It's a text-based format, so what we'd probably do is pop up a text box with the OPLM code, and the user could copy/paste as needed.

Unfortunately, I don't see myself getting to this for a very long time, if ever. Pull requests welcome though!

**Comment by Willow-Systems** [comment]
Whilst OPML sounds good from an interoperability point of view, exporting as a JSON object would probably be much simpler to implement, there also appears to be [a few libraries](https://github.com/jairsjunior/json-to-opml) which can do JSON -> OPML.

Is the source for the configuration page public?

**Comment by Wowfunhappy** [comment]
>Is the source for the configuration page public?

Yes, the page is served right from this repo via Github Pages, in the docs folder. :) That said, I would _really_ want any user-facing export to be OPML.

I also think #4 would go a long way towards making adding/removing feeds less painful.

**Comment by Cralex** [comment]
One current thing with importing from OPML is that the feed title size is currently limited to the number of characters the watch can display, so imported feed titles would either need to be truncated to fit, or the feed titles could be made to scroll if and when that gets figured out.
